test/scripts/withdraw.js

```diff
@@ -376,9 +376,10 @@ contract("Withdraw script", function (accounts) {
         assert.equal(requestedWithdrawal, "0", "A withdrawal request is still pending")
       }
     })
-    it("claims only amounts worth > 1 USD", async () => {
+    /*
+    // TODO: Write a test to somehow check it doesn't try to withdraw from all brackets if only one side is funded?
```

Comment:
Not sure that this is desired, because even if only one side is funded (initially), there could be trades and the brackets could hold either token.